### PR TITLE
Sort facets in Magento1

### PR DESCRIPTION
**Pull request by Kagaminara**
This MR documents how to enable facet sorting in Magento 1.

Related changes: https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2766

### CODE REVIEW

**Review by mGGk-fr — APPROVED** [review]

File: versioned_docs/version-2.x/magento1/installation.mdx [hunk 1]
```diff
@@ -217,9 +219,30 @@ For more security add a random key on `frontcommerce_secret_key` in your
...
+If you want to be able to organize your facets' order, you will need to enable
+it in the `magentoProvider` (see
+[code](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/fcaf0d0ddec410caf1a4d250d197e5f382e9004b#d43f790dbbf8f5142f59364a4b017caa013c10b5_82_83)).
+In this provider, override `magento.attributes.useFacetSortOrder` to set it to
+`true`.
+
+:::tip
+
+To learn more about how to override configuration provider, see our
+[guide on overriding an existing provider](https://developers.front-commerce.com/docs/2.x/advanced/server/configurations#override-an-existing-configuration).
```
Comment (dpobel) [inline versioned_docs/version-2.x/magento1/installation.mdx:235]:
```suggestion
[guide on overriding an existing provider](/docs/2.x/advanced/server/configurations#override-an-existing-configuration).
```

File: versioned_docs/version-2.x/magento1/installation.mdx [hunk 2]
```diff
@@ -217,9 +219,30 @@ For more security add a random key on `frontcommerce_secret_key` in your
     // ...
 ```
 
+### Configure facet order
+
+<SinceVersion tag="2.19" />
```
Comment (dpobel) [inline versioned_docs/version-2.x/magento1/installation.mdx:224]: 2.19 ? is this planned to be backported ? Also, [according to this page](https://www.notion.so/frontcommerce/Releases-a345faecf6624ce6bd91623f8580b16e?pvs=4#d9420f41fe5749d48b652318cdd257e0), 2.19 support ended with 3.0.0 if so, this should not be a `feat` commit and this MR should not target `next` but `main`

File: versioned_docs/version-2.x/magento1/installation.mdx [hunk 3]
```diff
@@ -217,9 +219,30 @@ For more security add a random key on `frontcommerce_secret_key` in your
...
+
+:::
+
+Once that's done, all facets will be sorted by their "position" attribute that
+is defined in the Magento's back office.
+
 ### Clean Magento cache
 
 ## Ensure it works Once this is done, all the checks should be green in your
+
```
Comment (dpobel) [inline versioned_docs/version-2.x/magento1/installation.mdx:245]: this page content is bit broken (too much content in headings) but adding this line break makes it worse

File: versioned_docs/version-2.x/magento1/installation.mdx (hunk 2 above)
Comment (Kagaminara) [reply versioned_docs/version-2.x/magento1/installation.mdx:224]: Nope, it was a copy-paste fail...

File: versioned_docs/version-2.x/magento1/installation.mdx (hunk 3 above)
Comment (Kagaminara) [reply versioned_docs/version-2.x/magento1/installation.mdx:245]: Yeah I'm not sure why there's a line break here. I think that should be 
```
## Ensure it works

Once this is done blblablab
```

Will fix.

File: versioned_docs/version-2.x/magento1/installation.mdx (hunk 2 above)
Comment (Kagaminara) [reply versioned_docs/version-2.x/magento1/installation.mdx]: Fixed

**Review by dpobel — APPROVED** [review]

**Pull request merged**